server/php/settings.php

```diff
@@ -0,0 +1,49 @@
+<?php
+return [
+    'settings' => [
+        'displayErrorDetails' => true, // set to false in production
+        'addContentLengthHeader' => false, // Allow the web server to send the content-length header
+
+        // Monolog settings
+        'logger' => [
+            'name' => 'slim-app',
+            'path' => isset($_ENV['docker']) ? 'php://stdout' : __DIR__ . '/logs/app.log',
+            'level' => \Monolog\Logger::DEBUG,
+        ],
+
+        'stripe' => [
+          // You shouldn't have to touch this
+          'apiVersion' => '2019-02-11',
```

Comment:
Any reason for this not to be the latest version `2019-03-14` as of right now?

Reply:
No reason, just bumped the version.